### DISCUSSION TITLE
Minimum requirements for Bioconductor events

### DISCUSSION THREAD

**Discussion by kozo2** [BioconductorBoards/CABWorkingGroups#6] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to discuss the minimum requirements for Bioconductor events. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.

  To get started, comment below and tell us about your idea (for the minimum requirements for Bioconductor events).

**Comment by kozo2** [comment]
- Ensure good gender balance of invited speakers
- Code of Conduct

**Comment by aedin** [comment]
- have a bioconductor focus or be of interest to bioconductor users/developers
- Ensure all participants have access to read and accept the code of conduct.  Have a mechanism in place to report and address CoC issues.  

Things to thing about
- Is there a requirement to have a member of the bioconductor community in the organizing committee
- What is the event is local (within institute, or local to an area... eg the Boston Bioconductor meetup)?
- Provide advance notice (what is the advance notice period)
- Do we just want non-profit events or are Commercial events included?
- What about social or after conference events.

**Reply by lgatto** [reply]
Re

> Is there a requirement to have a member of the bioconductor community in the organizing committee

May be that could be an optional requirement that is at the appreciation of the Bioconductor community (CAB I presume) upon inspection of the event organisation request?

**Reply by kozo2** [reply]
@aedin 
> Is there a requirement to have a member of the bioconductor community in the organizing committee

If we include this to the requirements, I think a Bioconductor member page should be set up on bioconductor.org (we call this BiocIdentity?)
so that everyone can see which community the person runs.

**Reply by kozo2** [reply]
> What is the event is local (within institute, or local to an area... eg the Boston Bioconductor meetup)?

I don't have a good idea to define it.
For the time being, I roughly divide it into Bioc and other events. Please refer to https://github.com/BioconductorBoards/CABWorkingGroups/blob/main/minimum_requirements_for_events.md

**Reply by kozo2** [reply]
> Provide advance notice (what is the advance notice period)

For the time being, I decided to do it **4months** before for Bioc and **2weeks** before for non-Bioc events.
(See
https://github.com/BioconductorBoards/CABWorkingGroups/blob/main/minimum_requirements_for_events.md)

Let me know if you have any opinions.

**Reply by kozo2** [reply]
> Do we just want non-profit events or are Commercial events included?

I have no experience with commercial events, so I ask others to comment on this on http://community-bioc.slack.com/ .

**Reply by kozo2** [reply]
> What about social or after conference events.

There seems to be something that does that kind of thing in virtual space these days, but I haven't been able to investigate it yet.
(I will investigate it and take notes.)

**Reply by jorainer** [reply]
> @aedin
> 
> > Is there a requirement to have a member of the bioconductor community in the organizing committee
> 
> If we include this to the requirements, I think a Bioconductor member page should be set up on bioconductor.org (we call this BiocIdentity?)
> so that everyone can see which community the person runs.

Suggestion: the main organizer should present the event briefly in a CAB meeting where it could be discussed if and how CAB (members) can help in organization or should be part of the organizing committee. Such things could e.g. also depend on the size of the event etc.
